.github/workflows/build-status-check.yml

```diff
@@ -33,7 +33,7 @@ jobs:
 
   build_samples_project:
     name: Build Samples
-    uses: KinsonDigital/Infrastructure/.github/workflows/build-csharp-project.yml@v13.0.1
+    uses: KinsonDigital/Infrastructure/.github/workflows/build-csharp-project.yml@v13.1.0
```

Comment:
Need to add the `checkout-ref` and `checkout-repository` workflow inputs.
Make sure to use Velaptor as a guide.  We implemented it wrong with the CASL project.

Reply:
duplicated?